encoding/generic.go

```diff
@@ -3,7 +3,8 @@ package encoding
 import "fmt"
 
 type FormatOptions struct {
-	Strict bool `mapstructure:"strict,omitempty"`
+	Strict    bool `mapstructure:"strict,omitempty"`
+	MaxLength int  `mapstructure:"max_length,omitempty"`
```

Comment:
This comes in as part of the configuration parameters? If so can you add something to the docs about it?

Reply:
There is no doc about the input conf 😢

Reply:
https://github.com/criteo-forks/carbon-relay-ng/blob/master/docs/input.md or https://github.com/criteo-forks/carbon-relay-ng/blob/master/docs/config.md we could add it